lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -322,19 +322,14 @@ def _monitor(self, poll_interval):
                                   now.strftime("%H:%M:%S"))
                 self._poll_stats(datapath)
 
-    def _poll_stats(self, datapath):
+    def _poll_stats(self, datapath, cookie = 0, cookie_mask = 0):
```

Comment:
In general, let's try to type annotate as much as possible. 
[cheat sheet for type annotation](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html)